go/stellar/loader.go

```diff
@@ -255,18 +256,17 @@ func (p *Loader) uiPaymentInfo(m libkb.MetaContext, summary *stellar1.PaymentLoc
 		StatusDescription: summary.StatusDescription,
 	}
 
-	// calculate the payment delta
-	username := p.G().ActiveDevice.Username(m)
-	if msg.sender.Eq(username) {
-		info.Delta = stellar1.BalanceDelta_DECREASE
-		// check if sending to self
-		if summary.TargetType == stellar1.ParticipantType_KEYBASE {
-			if libkb.NewNormalizedUsername(summary.Target).Eq(username) {
-				info.Delta = stellar1.BalanceDelta_NONE
-			}
-		}
+	info.Delta = stellar1.BalanceDelta_NONE
+
+	// Calculate the payment delta
+	if summary.FromType == stellar1.ParticipantType_OWNACCOUNT {
```

Comment:
do you also want to check if `summary.ToType == OWNACCOUNT`?

Reply:
"Either neither or both are OWNACCOUNT". But sure might as well. Faith in comment invariants and all that